### DISCUSSION TITLE
Possibility to get one merge request for docker images and helm charts.

### DISCUSSION THREAD

**Discussion by dr460neye** [renovatebot/renovate#20687] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Hi there,

as i could not figure out how to have one merge request for multiple managers, i thought i will ask here how to make it possible or, when this is not possible yet, to discuss if a feature would make sense.

The current setup:

- a mono repo containing all helm charts which will get deployed
- Images are in a proxy registry, so all helm charts have an overwritten registry entry.

Current behaviour is, that 2 different merge requests will be created. One for the docker image update, one for the helm chart.

As the images need to be provided to a second registry first, it complicates my current scenario, as image copies are managed by a different repo.
In my case, i will take out this special case.

**Comment by rarkins** [comment]
Use packageRules with groupName and one or more match* filters. Eg matchManagers or matchDatasources

**Reply by dr460neye** [reply]
Wouldnt this mean i need to manually configure one group per helm chart? As this would be unrealistic by 200 charts in use.

**Reply by rarkins** [reply]
I guess I didn't understand your description then (still don't).

Ultimately branches are determined by `depName`, so that would need to be the same if you want multiple images updated together, and if it's not the same then you'd need to define `groupName` using package rules

**Reply by dr460neye** [reply]
I have a branch, containing multiple "bundles". They are represented by one folder.
This folder contains a helm chart ( Charts.yaml ) dependency. The related dependency contains a docker image.
At the moment i receive two different merge requests. One for the helm chart, one for the docker image url (  values.yaml )

I want to achieve, that i receive a single MR, containing the "Chart.yaml" update and in parallel thee "values.yaml" update. 
As i receive two different merges, automerge is not possible. 

I see the option with folder based rules, but this would mean i need to create a single package rule for every single folder. This is not maintainable, so i look for automated solutions. e.g. autogrouping by "all in the same folder" etc.

**Reply by rarkins** [reply]
Look up the `packageFiles with updates` debug log message and then locate two such packages which you'd like to update together, but which currently are updated separately. Copy paste the entries for those two packages here
